deploy/examples/monitoring/grafana.yaml

```diff
@@ -0,0 +1,266 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-grafana
+  namespace: rook-ceph
+  labels:
+    app: grafana
+spec:
+  storageClassName: ceph-storage
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-ini
+  namespace: rook-ceph
+  labels:
+    app: grafana
+data:
+  grafana.ini: |
+    [server]
+      protocol = https
+      http_port = 3000
+      cert_file = /etc/grafana/certs/tls.crt
+      cert_key  = /etc/grafana/certs/tls.key
+    [analytics]
+      reporting_enabled = false
+      check_for_updates = true
+    [log]
+      mode = console
+      level = info
+    [security]
+      allow_embedding = true
+    [auth.anonymous]
+      enabled = true
+      org_name = Main Org.
+      org_role = Viewer
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-datasources
+  namespace: rook-ceph
+  labels:
+    app: grafana
+data:
+  prometheus.yaml: |-
+    {
+        "apiVersion": 1,
+        "datasources": [
+            {
+               "access":"proxy",
+                "editable": true,
+                "name": "Dashboard1",
+                "orgId": 1,
+                "type": "prometheus",
+                "url": "PROMETHEUS-SVC",
+                "version": 1
+            }
+        ]
+    }
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboards
+  namespace: rook-ceph
+  labels:
+    app: grafana
+data:
+  ceph-dashboard.yml: |-
+    apiVersion: 1
+    providers:
+      - name: 'Dashboard1'
+        torgId: 1
+        folder: 'ceph-dashboard'
+        type: file
+        disableDeletion: false
+        updateIntervalSeconds: 3
+        editable: false
+        options:
+          path: '/etc/grafana/dashboards/ceph-dashboard'
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-tls-cert
+  namespace: rook-ceph
+  labels:
+    app: grafana
+type: kubernetes.io/tls
+data:
+  tls.crt: |
```

Comment:
Let's not ship a secret, even as an example. Can we tell them to generate the cert instead?